README.md

```diff
@@ -1,3 +1,81 @@
+<<<<<<< HEAD
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+=======
 # Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
- [ ] It would be better to fix your README file. Please follow the next steps: ♻️ 

- Line 1 --> remove `<<<<<<< HEAD`.
- Line 4 --> Add the name of your project.
- Line 6 --> Add the description of your project.
- Line 8 --> Add a right link to a screenshot of the project. If you don't have one, please, remove it until you have one ready.
- Line 12 to 16 --> Add the right technologies that you are using for your project.
- Line 20 --> Add a right link to the `live demo link` of the project. If you don't have one, please, remove it until you have one ready.
- Line 25 to 26 --> remove comments that are not necessary.
- Line 33 to 41 --> Please, write the steps or commands to run your project locally.
- Line 47 to 57 --> You are using links that don't belong to you. Please, use your own links.
- Line 79 to 149 --> It would be better to remove this part because is the README that comes from Create React App but if you want to use them. Please, try to move them to the right place where it makes sense.

Reply:
Modified Readme File. Fixed!!!